prehook/richcontainer/richcontainer_hook.go

```diff
@@ -21,11 +31,16 @@ func init() {
 			if !isRichMode(spec) {
 				return nil
 			}
+
 			launcher, err := getRichModeLauncher(spec)
 			if err != nil {
 				return err
 			}
 
+			if err = commonHook(opt, spec); err != nil {
```

Comment:
I am wondering if we name this function in `commonHook`? I do not think the function details did a common hook? Could you give more explanation about that? @wangforthinker